src/org/zaproxy/zap/extension/httpsinfo/HttpsInfoOutputPanel.java

```diff
@@ -1,342 +1,274 @@
-				cs.append(NEWLINE);
-				csMap.put(cipher, cipher);
-			}
-		}
-		cipherSuites.setText(cs.toString());
-		cipherSuites.setCaretPosition(0);
-	}
-
-	public void addComponentsToPane(Container pane) {
-		pane.setLayout(new GridBagLayout());
-		GridBagConstraints c = new GridBagConstraints();
-		c.fill = GridBagConstraints.BOTH;
-		c.gridwidth = 1;
-		c.weightx = 1.0;		
-		c.weighty = 0.5;
-		
-		c.gridx = 0;
-		c.gridy = 0;
-		c.anchor = GridBagConstraints.NORTH;
-		pane.add(new JScrollPane(general), c);
-		
-		c.gridx = 0;
-		c.gridy = 1;
-		c.gridwidth = 3;
-		c.anchor = GridBagConstraints.SOUTH;
-		pane.add(new JScrollPane(cipherSuites), c);
-	}
-
-	/**
-	 * Check if ZAP is configured to use an outbound proxy. If it is then warn via a GUI dialog. 
-	 * Results may be inaccurate, representing the connection to the proxy instead of the 
-	 * connection to the target.
-	 */
-	private void checkProxyChainEnabled() {
-		if (Model.getSingleton().getOptionsParam().getConnectionParam().isUseProxyChain()) {
-			String warningMsg = MessageFormat.format(
-					Constant.messages.getString("httpsinfo.warn.outgoing.proxy.enabled"),
-					Constant.messages.getString("httpsinfo.name"));
-			View.getSingleton().showWarningDialog(warningMsg);
-		}
-	}
-	
-	/**
-	 * The following raise__Alert methods are being left in the code base for the time being.
-	 * Though DeepViolet does not currently have checks for Beast and Crime they may come back, or be
-	 * otherwise implemented.
-	 */
-	
-	@SuppressWarnings("unused")
-	private void raiseBeastAlert() {
-		Alert alert = new Alert(BEAST_PLUGIN_ID, Alert.RISK_INFO, Alert.CONFIDENCE_MEDIUM,
-				Constant.messages.getString("httpsinfo.beast.name"));
-		alert.setDetail(Constant.messages.getString("httpsinfo.beast.desc"), // Desc
-				getBaseMessage().getRequestHeader().getURI().toString(), // URI
-				null, // Param
-				null, // Attack
-				null, // OtherInfo
-				Constant.messages.getString("httpsinfo.beast.soln"), // Solution
-				Constant.messages.getString("httpsinfo.beast.refs"), // References
-				null, // Evidence
-				311, // CWE ID
-				4, // WASC ID
-				getBaseMessage()); // HTTPMessage
-		extensionAlert.alertFound(alert, getBaseMessage().getHistoryRef());
-	}
-
-	@SuppressWarnings("unused")
-	private void raiseCrimeAlert() {
-		Alert alert = new Alert(CRIME_PLUGIN_ID, Alert.RISK_LOW, Alert.CONFIDENCE_MEDIUM,
-				Constant.messages.getString("httpsinfo.crime.name"));
-		alert.setDetail(Constant.messages.getString("httpsinfo.crime.desc"), // Desc
-				getBaseMessage().getRequestHeader().getURI().toString(), // URI
-				null, // Param
-				null, // Attack
-				null, // OtherInfo
-				Constant.messages.getString("httpsinfo.crime.soln"), // Solution
-				Constant.messages.getString("httpsinfo.crime.refs"), // References
-				null, // Evidence
-				311, // CWE ID
-				4, // WASC ID
-				getBaseMessage()); // HTTPMessage
-		extensionAlert.alertFound(alert, getBaseMessage().getHistoryRef());
-	}
-	
-}
+/*

```

Comment:
This could keep the same line endings, would make the diff smaller (I think, some of the parts seem to be exactly the same).

Reply:
I had decided that since it was moving to a new class I could reformat it. I can try to revert a bunch of the formatting changes.

Reply:
I'd start by the line endings changes as that touches everything, the overall format looked very similar to before.

Reply:
I'm unable to change them.
![image](https://user-images.githubusercontent.com/7570458/37029677-4df95c0a-2106-11e8-80ca-dcdfc8525bcd.png)

Reply:
:/ I can push that change?

Reply:
Sure if it’s simple and you have a moment.

Reply:
Done (pushed a new commit just in case, hopefully that can be fixed up).